resources/lang/en/hint.php

```diff
@@ -317,9 +317,9 @@
         "Mire Shed - Left" => "is near Misery Mire",
         "Mire Shed - Right" => "is near Misery Mire",
         "Misery Mire - Big Chest" => "is in Misery Mire",
-        "Misery Mire - Big Key Chest" => ["is in Misery Mire", "is fire locked in Misery Mire"],
+        "Misery Mire - Big Key Chest" => ["is fire locked in Misery Mire", "is in Misery Mire"],
```

Comment:
why were these reordered?

Reply:
For multi-locations, LocationCollection::getHint returns the first entry, not a random entry: https://github.com/sporchia/alttp_vt_randomizer/blob/master/app/Support/LocationCollection.php#L116

Making it always say `X and Y is in Misery Mire` instead of `X and Y is fire locked in Misery Mire` seemed to be an unintentional side effect of the hint location standardization that took place in v31. The order shouldn't be important in Location::getHint for single-location hints because it fy_shuffles the array before choosing a descriptor: https://github.com/sporchia/alttp_vt_randomizer/blob/master/app/Location.php#L214

Only multi-location hints are affected by the order, specifically which entry is first. Since multi-location hints haven't been generated since the new hint description was added, nobody has been able to realize that adding the new hint description had an extra side effect.

It didn't seem intentional because right now the "annoying location" hints are the only way to get a multi-location hint so people will always know anyways that it's the two fire-locked chests in mire since the hint spoils two items instead of one (similar for left side swamp). Only new players who are still learning the randomizer won't know that it's always the two fire-locked items / two left-side items and they'll be unpleasantly surprised once they find that out.

This is what I was referring to in the third paragraph of #709